src/modules/launcher/PowerLauncher/ReportWindow.xaml.cs

```diff
@@ -44,6 +44,8 @@ private void SetException(Exception exception)
 
             StringBuilder content = new StringBuilder();
             content.AppendLine(ErrorReporting.RuntimeInfo());
+
+            // Using InvariantCulture since this is internal
             content.AppendLine($"Date: {DateTime.Now.ToString(CultureInfo.InvariantCulture)}");
```

Comment:
I believe this is displayed to the user in the report window. This should be current culture.